.env.example

```diff
@@ -23,7 +23,7 @@ GITHUB_TOKEN={fill me in}
 # BYPASS_DETAILS={optional 'Some text explaining bypass procedure'}
 # BUDDY_CHECK_FEATURE={optional, set to 1 to enable buddy_check feature; 0 to disable}
 
-# DEPLOY_MAX_MINUTES_PENDING={optional, set to 20 for max minutes a deploy is pending}
+# BUDDY_CHECK_TIME_LIMIT={optional, set to 20 for max minutes a deploy is pending}
```

Comment:
I was thinking it's a 'timeout', after a certain time it does not work anymore.
`BUDDY_CHECK_TIMEOUT={optional, default 20, how many minutes a commit can be redeployed without needing another buddy check}`